editors/vscode/client/extension.spec.ts

```diff
@@ -49,30 +50,29 @@ suite('commands', () => {
 
     const isEnabledAfter = workspace.getConfiguration('oxc').get<boolean>('enable');
     strictEqual(isEnabledAfter, false);
+
+    // enable it for other tests
+    await commands.executeCommand('oxc.toggleEnable');
+    await sleep(500);
```

Comment:
this feels like it could cause other tests to fail: what if we don't execute this part? maybe we could add a helper function to ensure that oxc is enabled, and execute that at the beginning of every test?

Reply:
> this feels like it could cause other tests to fail: what if we don't execute this part? m
Then the other tests have the vscode plugin enabled but the language server disabled and could probabbly fail (like this one)

> maybe we could add a helper function to ensure that oxc is enabled, and execute that at the beginning of every test?

Sure we need a test structure when creating more files and deleting them in the workspace folders and making sure everything is pre-configured like we expect.
This is future a goal of mine